cypress/integration/metaNewsSpec.js

```diff
@@ -54,4 +60,32 @@ describe('Article Meta Tests', () => {
       secondElement.should('have.attr', 'rel', 'dns-prefetch');
     });
   });
+
+  it('should have Facebook meta data', () => {
```

Comment:
In refinement sessions, we considered that we don't need to test specific metadata implementations. Has that now changed?

@jamesbrumpton @sareh

Reply:
You're correct that this was decided in the refinement sessions (that's why the issues mention unit tests only). I assumed that since James added cypress tests, his opinion on this had changed.  Since that's his domain, as the team's Tester, I thought we'd go with that.

Reply:
I don't disagree! Just looking for some more context before I review these tests closer :)

Reply:
Let's include them for now and if we feel they're unnecessary further down the line then I'm happy to discuss/remove them. Are we ok with that? :)

Reply:
Okay. There's a lot of duplication between `metaNewsSpec` and `metaPersianSpec`; the only thing that's changing between them is the content being tests and routes that are hit. We could do something similar to the [routing specs](https://github.com/bbc/simorgh/blob/latest/cypress/integration/routingSpec.js) that uses this [fixture data](https://github.com/bbc/simorgh/blob/latest/cypress/support/testData/services.js). Also similar to [this cypress guide](https://docs.cypress.io/guides/core-concepts/writing-and-organizing-tests.html#Dynamically-Generate-Tests).